setup.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Ok, I just read the comment for the commit. I'm still -1 on adding this. You could install `tox` on Python 3 instead and do away with this problem that way.